cpp/src/arrow/util/bit-util.h

```diff
@@ -329,6 +327,30 @@ static inline void SetBitTo(uint8_t* bits, int64_t i, bool bit_is_set) {
                  kBitmask[i % 8];
 }
 
+static inline void SetBitsTo(uint8_t* bits, int64_t i_begin, int64_t i_end,
```

Comment:
Add a docstring?

Reply:
Another concern: other bit-util APIs take `start, length` arguments, taking `start, end` here makes things a bit confusing and error-prone.